### ISSUE TITLE
Menu "Recent" does not show any recently-viewed items

### ISSUE THREAD

**Issue by mariav0** [civicrm/org.civicrm.recentmenu#15]
I've tried release 1.5 and the master-version of this extension but it does not show me any entries.
Current CiviCRM version: 5.51.1
CMS: Wordpress
Does anyone else has this problem, too?

**Comment by mariav0** [comment]
The console shows me a 403 Forbidden status:

![image](https://user-images.githubusercontent.com/70949178/195347072-26a55be8-1794-4dd7-9ef3-634cbb9c15fb.png)

**Comment by colemanw** [comment]
@mariav0 this sounds like a permissions problem. Does your user have "access ajax api" permission?

**Comment by mariav0** [comment]
@colemanw Thanks for your quick response. I am not sure which permission it should be since I have the administrator role. The permission "access ajax api" is even granted for anonymous users.

**Comment by colemanw** [comment]
Actualy @mariav0 now that I look at your screenshot, it appears the `recentmanu.js` file is not loading. Have you had this problem with any other extensions? The path to the file looks correct.

**Comment by mariav0** [comment]
No, all other extensions work fine. Do you think of a reason why it does not load?
